datastore/longhorn.go

```diff
@@ -3404,6 +3404,19 @@ func (s *DataStore) IsNodeSchedulable(name string) bool {
 	return nodeSchedulableCondition.Status == longhorn.ConditionStatusTrue
 }
 
+func (s *DataStore) IsNodeHasDiskUUID(nodeName, diskUUID string) (bool, error) {
+	node, err := s.GetNodeRO(nodeName)
+	if err != nil {
+		return false, err
+	}
```

Comment:
What will happen if the node is not found? If the node is not found, does it mean the disks are gone, the it need to return `false, nil`.